logdna/configs.py

```diff
@@ -3,5 +3,8 @@
     'MAX_LINE_LENGTH': 32000,
     'FLUSH_INTERVAL': 5,
     'FLUSH_BYTE_LIMIT': 2 * 1024 * 1024,
-    'LOGDNA_URL': 'https://logs.logdna.com/logs/ingest'
+    'LOGDNA_URL': 'https://logs.logdna.com/logs/ingest',
+    'BUFFER_BYTE_LIMIT': 4194304,
```

Comment:
I'm not sure this is necessary - I can't see how it would ever be triggered before `FLUSH_BYTE_LIMIT` applies, but if we do keep it, let's format it like `4 * 1024 * 1024` for readability.

Reply:
right. I will take it away. I just initially wanted to update the current buf size according to the flush size and have it zero in the case of exception, so the buffer can take new lines up to the flush limit but also keep the lines from the failed call.

Reply:
No Actually I won't take it away, I will correct the logic.